packages/hash/frontend/src/pages/index.page.tsx

```diff
@@ -17,20 +21,28 @@ const Page: NextPageWithLayout = () => {
     if (user) {
       // Temporarily redirect logged in user to their account page
       void router.push(`/${user.accountId}`);
-    } else {
+    }
+
+    if (!kratosSession) {
       void router.push("/login");
     }
-  }, [loading, router, user]);
+  }, [loading, router, user, kratosSession]);
 
   return (
-    <Box component="main" pt="30vh" display="flex" justifyContent="center">
-      <Typography
-        variant="h1"
-        sx={({ palette }) => ({ color: palette.gray[70], fontWeight: 400 })}
-      >
-        Loading...
+    <Container sx={{ pt: 10 }}>
+      <Typography variant="h1" gutterBottom>
+        {kratosSession
+          ? "You have a kratos session"
+          : "You don't have a kratos session"}
+      </Typography>
+      <Button onClick={logout}>Log out</Button>
+      <Typography gutterBottom>
+        This is what your kratos session looks like:
```

Comment:
I'm assuming this is very temporary? Was going to ask if we want to put a todo but probably not necessary

Reply:
Yeah, this is temporary. I was finding it helpful to have this information whilst testing/developing locally. Reason I checked it into git is because it'll probably also be useful for some of the upcoming PRs as well.

Of course this should be removed at some stage though, so have added a `@todo` in https://github.com/hashintel/hash/pull/889/commits/148ba915ad2d539890d6ee1ac32668bd1e06dcae

Reply:
Cool, cheers :)